### PR TITLE
update FAQ to point to new services doc

### PR DESCRIPTION
**Pull request by sethladd**
cc @qchong

### CODE REVIEW

File: faq.md [hunk 1]
```diff
@@ -140,9 +140,17 @@ we encourage you to email our mailing list:
...
-Yes. Flutter gives developers out-of-the-box access to some services
-and APIs from the mobile operating system. You can learn more about this
-at [[docs coming soon]]. If you want to do this, we encourage you to email our
+Yes. Flutter gives developers out-of-the-box access to _some_ services
+and APIs from the operating system. However, we want to avoid the
+"lowest common denominator" problem with most cross-platform APIs, so we
+do not intend to build cross-platform APIs for all native services and APIs.
+
+We encourage developers and the ecosystem to
+[access platform APIs with our service plugins](/platform-services/). Developers
```
Comment (qchong) [inline faq.md]: @sethladd The /platform-services guide being linked to does not currently mention "service plugins" (I believe this is internal only). You could instead say something like "We encourage developers to use Flutter's asynchronous message passing system to create your own integrations with platform and third-party APIs."

Comment (sethladd) [inline faq.md]: I like it. Done.

**Comment by sethladd** [comment]
@qchong ptal thanks!

**Comment by qchong** [comment]
lgtm, thanks for making the changes!

**Pull request merged**